tso_session/intro_TSO_data_pipeline.ipynb

```diff
@@ -136,11 +136,74 @@
     "First the imports.\n"
```

Comment:
<p>All imports should happen at the beginning of the notebook. If it's important to highlight that these are specific to Stage 2, you can mention so in the import cell with a comment above them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>done</p>